src/coreComponents/constitutive/fluid/multifluid/CO2Brine/CO2BrineFluid.cpp

```diff
@@ -137,6 +138,48 @@ integer CO2BrineFluid< PHASE1, PHASE2, FLASH >::getWaterPhaseIndex() const
   return PVTFunctionHelpers::findName( m_phaseNames, expectedWaterPhaseNames, viewKeyStruct::phaseNamesString() );
 }
 
+template< typename PHASE1, typename PHASE2, typename FLASH >
+void CO2BrineFluid< PHASE1, PHASE2, FLASH >::checkTablesParameters( real64 const pressure,
+                                                                    real64 const temperature ) const
+{
+  if( !m_checkPVTTablesRanges )
+    return;
```

Comment:
```suggestion
  if( !m_checkPVTTablesRanges )
  {
    return;
  }
```
sorry for the nitpicking, this is to stick to our style convention